src/components/Home/PoliSearchBar.tsx

```diff
@@ -38,7 +38,7 @@ export const PoliSearchBar: FC<{
         <Animated.View
             style={{
                 marginTop: 46,
-                marginBottom: 23,
+                marginBottom: 15,
```

Comment:
non dovrebbe essere 18?
```suggestion
                marginBottom: 18,
```
da spec di figma lo spazio tra la barra di ricerca e gli elementi del menu è di 31px
15 margin bottom della SearchBar + 8 margin top dei MenuButton + 5 margin top del menu = 28

Reply:
si mea culpa, ammetto di non aver controllato ma di essere andato ad occhio